services/data-pipeline-dashboard/README.md

```diff
@@ -7,7 +7,6 @@ Much of this is copied from the [can-scrapers prefect README](https://github.com
 - SSH into it
 - `sudo apt-get install --yes git git-lfs nginx`
 - `git clone https://github.com/covid-projections/covid-data-model`
-- `git clone https://github.com/covid-projections/covid-data-public`
```

Comment:
The dashboard isn't really working right now, but I think it shouldn't be necessary to clone `covid-data-public`, as all of that code now lives in `covid-data-model`